docs/index.md

```diff
@@ -1,3 +1,134 @@
+<!-- vim: set textwidth=110: -->
+
 # rocd
 
-Welcome to rocd documentation!
+We start from a small preview of a system as a whole to understand what is the role of `rocd` component. The
+purpose of the [Roc Toolkit](https://roc-streaming.org/toolkit/docs/about_project/overview.html) (further
+referred to as `roc`) is to implement real-time audio streaming over network.
+
+## Entities
+
+Here are the top-level components:
+
+**host**
+:   Computer where the `roc` instance is currently running on. Computer platform may be on of:
+
+    - PC (personal computer, usually a desktop device with monitor or a similar output device),
+    - Android,
+    - iOS (future plans),
+    - embedded (is actually the same as PC, but usually doesn't have a monitor).
```

Comment:
1. Let's list PC platforms: Linux and other *nix, macOS, Windows (future plans).
2. I'd replace "embedded" with "embedded Linux", because for now we support only that.
3. I'd relax definition here: "is actually the same as PC, but usually doesn't have a monitor" => "*can be similar* to PC, but *often* doesn't have a monitor"; because people use very different things, e.g. yocto with squashfs or RPi with display and keyboard.
4. If we distinguish "embedded" from desktop, I think we also should distinguish "server". Unlike other options, server installation wouldn't have audio devices but can be used as proxy/transcoder/nat traversal server/etc.